packages/blockchain-link-utils/src/stellar.ts

```diff
@@ -0,0 +1,149 @@
+import { Horizon } from '@stellar/stellar-sdk';
+
+import { BigNumber } from '@trezor/utils/src/bigNumber';
+import type { Transaction, Target, TransactionDetail } from '@trezor/blockchain-link-types';
+
+const BASE_REVERSE = new BigNumber('5000000'); // 0.5 XLM, https://developers.stellar.org/docs/learn/fundamentals/lumens#base-reserves
+const ONE = new BigNumber('10000000');
+export const MINIMUM_RESERVE = ONE;
```

Comment:
Can those values be fetched? Historically on Ripple, they changed

Reply:
https://github.com/trezor/trezor-suite/pull/16483/files#diff-c6433311ad799aae03f70ad6559021ac8ecbc172cad155ddb736fcb35e332270R30-R32